user_docs/en/changes.t2t

```diff
@@ -24,15 +24,11 @@ What's New in NVDA
 
 
 == Changes for Developers ==
+- The [NVDA API Announcement mailing list https://groups.google.com/a/nvaccess.org/g/nvda-api/about] was created. (#13999)
+-
 
 
 === Deprecations ===

```

Comment:
Perhaps this is appropriate @feerrenrut?

Note that this link will only work once the changes in this PR reach NVDA stable (e.g. 2022.4 is released).
We will have to keep the blurb and can't make this change until the subsequent release (e.g. 2023.1).

```suggestion
=== Deprecations ===
Please refer to [the developer guide https://www.nvaccess.org/files/nvda/documentation/developerGuide.html#API] for information on NVDA's deprecation process.
```